platform/nodejs/test.js

```diff
@@ -91,6 +75,35 @@ async function main() {
     if ( result !== 0 ) {
         console.log(engine.toLogData());
     }
+}
+
+async function main() {
+    try {
+        const result = await enableWASM();
+        if ( result !== true ) {
+            console.log('Failed to enable all WASM code paths');
+        }
+    } catch(ex) {
+        console.log(ex);
+    }
+
+    const engine = await StaticNetFilteringEngine.create();
+
+    await engine.useLists([
+        fetch('easylist').then(raw => ({ name: 'easylist', raw })),
+        fetch('easyprivacy').then(raw => ({ name: 'easyprivacy', raw })),
+    ]);
+
+    runTests(engine);
```

Comment:
We're running the tests thrice now: with the original raw lists, then with no lists, and finally with lists loaded from serialized data. The first and third should give the same results. It's possible to verify this too, but I didn't go so far.